frontend/src/features/public-form/components/FormPaymentPage/stripe/components/PaymentItemDetailsBlock.tsx

```diff
@@ -0,0 +1,35 @@
+import { Box, Text } from '@chakra-ui/react'
+
+import { FormColorTheme } from '~shared/types'
+
+import { centsToDollars } from '~utils/payments'
+
+export interface PaymentItemDetailsBlockProps {
+  paymentItemName: string | undefined
+  colorTheme: FormColorTheme
+  paymentAmount: number | undefined
+}
+
+export const PaymentItemDetailsBlock = ({
+  paymentItemName,
+  colorTheme,
+  paymentAmount,
+}: PaymentItemDetailsBlockProps): JSX.Element => {
+  return (
+    <Box
+      backgroundColor={`theme-${colorTheme}.100`}
+      borderWidth="1px"
+      borderColor={`theme-${colorTheme}.300`}
+      borderRadius="4px"
+      p="0.7rem"
+      mb="2rem"
```

Comment:
nit: the gap caused by `mb="2rem"` here a bit ugly for the payment modal haha
<img width="597" alt="Screenshot 2023-05-03 at 2 11 25 PM" src="https://user-images.githubusercontent.com/37061143/235843585-daedd43c-0120-45fd-8a50-110b00647e8c.png">
Maybe we can take `mb` off the component and set `mb` at whatever parent component calls it?

Reply:
done! [06ff933](https://github.com/opengovsg/FormSG/pull/6235/commits/06ff933c1e6599fc07790d09e979fd3b45a152e5)